temboardagent/plugins/statements/__init__.py

```diff
@@ -11,6 +11,37 @@
 routes = RouteSet(prefix=b"/statements")
 
 
+query = """\
+SELECT
+  rolname AS username,
+  datname AS dbname,
+  queryid,
+  query,
+  calls,
+  total_time,
+  min_time,
+  max_time,
+  mean_time,
+  stddev_time,
+  rows,
+  shared_blks_hit,
+  shared_blks_read,
+  shared_blks_dirtied,
+  shared_blks_written,
+  local_blks_hit,
+  local_blks_read,
+  local_blks_dirtied,
+  local_blks_written,
+  temp_blks_read,
+  temp_blks_written,
+  blk_read_time,
+  blk_write_time
+FROM pg_stat_statements
+JOIN pg_authid ON pg_stat_statements.userid = pg_authid.oid
+JOIN pg_database ON pg_stat_statements.dbid = pg_database.oid
+"""
+
```

Comment:
What about returning pg_stat_statements.*? I don't think it hurts to have `dbid` and `userid`.

Reply:
I'd be in favor of keeping the `rolname` and `datname` original names.
@julmon opinion?

Reply:
Agreed with keeping `rolname` and `datname`. About returning `pg_stat_statements.*`, keep in mind the definition of this view could differ depending on Postgres version.

Reply:
@dlax then let's make sure we return a given list of fields for pg_stat_statements.